HHS/Head-Start-TTADP
# [TTAHUB-1837] Add collaborator permission check

## Description of change

Collaborators need the ability to view all events in their region. In addition they need to view and edit any reports they collaborate on. This change modifies the service call to get the appropriate events for the user.

## How to test

- If a user has read/write training reports they should see events for all regions and be able to edit them.
- If a user is a collaborator they should see all reports in their region and be able to edit any report they collaborate on.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1837


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
